core/src/test/resources/sql_schema/postgres_array_types.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE ArrayTypes (
+                            nonArrayColumn integer NOT NULL,
+                            arrayColumn integer[] NOT NULL,
+                            matrixColumn integer[][] NOT NULL,
+                            spaceColumn integer[][][] NOT NULL,
+                            manyDimensionsColumn integer[][][][] NOT NULL,
+                            exactSizeArrayColumn integer[3] NOT NULL,
+                            exactSizeMatrixColumn integer[3][3] NOT NULL
```

Comment:
would be good to have at least one case of something different from integer, like a text array

Reply:
there is another test case using string arrays. The table is StringArrayTable.